scripts/crossgen/crossgen_roslyn.cmd

```diff
@@ -1,15 +1,15 @@
+@echo off
+
```

Comment:
I don't think so - this reorder will cause this to be seen on the screen:
```
My\Path> REM Comment
My\Path> REM Comment
```
You would need to either go with @REM instead of REM or reorder back